spec/oas3.patched/components/schemas/Device.yaml.rej

```diff
@@ -0,0 +1,11 @@
+--- spec/oas3.patched/components/schemas/Device.yaml
```

Comment:
Note: device storage patch has been upstreamed and needs to be removed from this repo